gradle.properties

```diff
@@ -0,0 +1 @@
+excludeIntegrationTests=netezza,sqlserver,teradata
```

Comment:
I guess I we could use the basic `:shell` provisioner too. A simple `echo` would probably do it!